src/mergeProp.js

```diff
@@ -0,0 +1,28 @@
+import { curry } from 'ramda';
+import mergePath from './mergePath';
+
+/**
+ * Merges user provided data (subject) with some object (target)
+ * inside other object under corresponding property.
+ *
+ * @func mergeProp
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/1.18.0|v1.18.0}
+ * @category Object
+ * @sig [k] -> {a} -> {k: {a}} -> {k: {a}}
+ * @see {@link RA.mergeProp|mergeProp}
```

Comment:
This links to it self. Which doesn't make sense. Let's link to `mergeProps` and `mergePath`

```javascript
@see {@link RA.mergePath|mergePath}, {@link RA.mergeProps|mergeProps}